cmd/kube-proxy/app/conntrack.go

```diff
@@ -38,7 +36,7 @@ var readOnlySysFSError = errors.New("ReadOnlySysFS")
 
 func (realConntracker) SetMax(max int) error {
 	glog.Infof("Setting nf_conntrack_max to %d", max)
-	if err := sysctl.New().SetSysctl("net/netfilter/nf_conntrack_max", max); err != nil {
+	if err := sysfs.New().WriteInt("net/netfilter/nf_conntrack_max", max); err != nil {
```

Comment:
The argument here is a sysctl-ish name, not a sysfs file path.  Below you make the same call but with absolute paths to /sys/... - I don't know which is correct.